packages/components/src/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+	__experimentalText as Text,
+	__experimentalHStack as HStack,
+	Button,
+	VisuallyHidden,
+} from '@wordpress/components';
+import { useResizeObserver, useMergeRefs } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { useState, forwardRef, useRef } from 'react';
+import Menu from '../menu';
+import { BreadcrumbProps, BreadcrumbItemProps } from './types';
+import './style.scss';
+
+function BreadcrumbsMenu( { items }: { items: BreadcrumbItemProps[] } ) {
+	return (
+		<span className="breadcrumb__item-wrapper">
+			<Menu placement="bottom-start">
+				<Menu.TriggerButton
+					render={
+						<Button
+							size="compact"
+							className="breadcrumb__item"
+							text="…"
+							label={ __( 'More breadcrumb items' ) }
+						/>
+					}
+				/>
+				<Menu.Popover>
+					{ items.map( ( item, index ) => (
+						<Menu.Item
+							key={ `${ item.label }-${ index }` }
+							render={
+								<a href={ item.href } onClick={ item.onClick } className="breadcrumb__item" />
+							}
+						>
+							<Menu.ItemLabel>{ item.label }</Menu.ItemLabel>
+						</Menu.Item>
+					) ) }
+				</Menu.Popover>
+			</Menu>
+		</span>
+	);
+}
+
+function BreadcrumbItem( { item: { label, href, onClick } }: { item: BreadcrumbItemProps } ) {
+	return (
+		<span className="breadcrumb__item-wrapper">
+			<a href={ href } onClick={ onClick } className="breadcrumb__item">
```

Comment:
Should we render a `span` if the `href` is not defined or empty?

Reply:
Actually it makes no sense for a breadcrumb item not to have an `href`. I'll update the prop to be required for now, but if design thinks this should also work for buttons too, we should update the code to render different tags conditionally. --cc @jameskoster @jasmussen

Reply:
@ntsekouras are you asking whether the breadcrumb should support using Buttons for the trail items as well as links? I can't really think of a need for that. Apologies if I'm mis-interpreting the question.

Reply:
That was exactly my question. We can move forward with link then and revisit if we even need buttons in the future.